packages/salesforcedx-vscode-core/package.json

```diff
@@ -786,6 +786,11 @@
           "type": "boolean",
           "default": false,
           "description": "%experimental_deploy_retrieve_description%"
+        },
```

Comment:
I think this should be moved to `salesforcedx-vscode-lwc/package.json` since all the local dev configs live there.